pkg/updater/updater.go

```diff
@@ -432,3 +353,71 @@ func checkAvailableDiskSpace(fsDisk disk, paths ...string) error {
 	}
 	return nil
 }
+
+type requestKey int
+
+var requestStateKey requestKey
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏For my general understanding, will this work if there's another 0 somewhere in the context?

Reply:
Yep, it uses the type rather than the value: https://go.dev/blog/context#TOC_3.2.